.circleci/config.yml

```diff
@@ -19,11 +19,9 @@ jobs: # basic units of work in a run
           keys:
             - v1-pkg-cache
 
-      - run: make get_tools
-      - run: dep ensure -v
-      - run: go get -u github.com/kardianos/govendor
-      - run: govendor fetch -tree github.com/ethereum/go-ethereum/crypto/secp256k1
+      - run: make tools
       - run: make install
+      - run: make lint
```

Comment:
no need to run `make lint` here, will enable golangci bot on this repo, @jackzampolin could you enable it

Reply:
we added golangci so now it should fire